app/assets/stylesheets/admin.scss

```diff
@@ -1171,12 +1171,45 @@ code {
       position: absolute;
       border-radius: 50%;
     }
+
+    .map-icon-related {
+      background: $brand;
+    }
+  }
+
+  .layer-related-icon {
+    display: inline-block;
+    width: 15px;
```

Comment:
How about reusing styles? Here's a possible patch (warning: haven't tested it yet :sweat_smile:).

```diff
diff --git a/app/assets/stylesheets/admin.scss b/app/assets/stylesheets/admin.scss
index be6ab1f5a2..e7b0706ccd 100644
--- a/app/assets/stylesheets/admin.scss
+++ b/app/assets/stylesheets/admin.scss
@@ -1147,6 +1147,24 @@ code {
   width: 100%;
   height: 350px;
 
+  @mixin marker-icon($width) {
+    width: $width;
+    height: $width;
+    background: #00cae9;
+    border-radius: 50% 50% 50% 0;
+    transform: rotate(-45deg);
+
+    &::after {
+      background: #fff;
+      border-radius: 50%;
+      content: "";
+      height: $width / 2 - 1;
+      margin: #{$width / 4} 0 0 #{$width / 4};
+      position: absolute;
+      width: $width / 2 - 1;
+    }
+  }
+
   .map-marker {
     visibility: visible;
     position: absolute;
@@ -1155,21 +1173,7 @@ code {
     margin-top: -5px;
 
     .map-icon {
-      width: 30px;
-      height: 30px;
-      border-radius: 50% 50% 50% 0;
-      background: #00cae9;
-      transform: rotate(-45deg);
-    }
-
-    .map-icon::after {
-      content: "";
-      width: 14px;
-      height: 14px;
-      margin: 8px 0 0 8px;
-      background: #fff;
-      position: absolute;
-      border-radius: 50%;
+      @include marker-icon(30px);
     }
 
     .map-icon-related {
@@ -1178,23 +1182,9 @@ code {
   }
 
   .layer-related-icon {
-    display: inline-block;
-    width: 15px;
-    height: 15px;
-    border-radius: 50% 50% 50% 0;
-    background: #00cae9;
-    transform: rotate(-45deg);
+    @include marker-icon(15px);
     background: $brand;
-  }
-
-  .layer-related-icon::after {
-    content: "";
-    width: 7px;
-    height: 7px;
-    margin: 4px 0 0 4px;
-    background: #fff;
-    position: absolute;
-    border-radius: 50%;
+    display: inline-block;
   }
 
   .map-attributtion {
```